client/src/js/references/components/ReferenceSelect.js

```diff
@@ -0,0 +1,35 @@
+import { map } from "lodash-es";
+import React from "react";
+import { ListGroup } from "react-bootstrap";
+import styled from "styled-components";
+import { NoneFound } from "../../base";
+import { ReferenceSelectItem } from "./ReferenceSelectItem";
+
+const ReferenceSelectList = styled(({ hasError, ...rest }) => <ListGroup {...rest} />)`
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['hasError' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/virtool/virtool/pullRequest?prid=4644906)